src/core/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -642,4 +642,7 @@
   <data name="About_PowerToys.Text" xml:space="preserve">
     <value>Microsoft PowerToys is a set of utilities for power users to tune and streamline their Windows experience for greater productivity</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="FancyZones_MultiMonitorModeCheckBoxControl.Content" xml:space="preserve">
+    <value>Run FancyZones in multi-monitor mode</value>

```

Comment:
Let's rename it to "Allow zones to span across monitors"

Reply:
Done, will commit soon